### ISSUE TITLE
Repository has diverged with sourcesup/renater FrontC git repository

### ISSUE THREAD

**Issue by jordr** [BinaryAnalysisPlatform/FrontC#53]
Hi,

I recently came to learn the existence of another FrontC repository maintained by the TRACES team/University of Toulouse, the original authors (Hugues Cassé).
The repository lives at https://git.renater.fr/anonscm/git/orange/Frontc.git (project is https://sourcesup.renater.fr/scm/?group_id=5026).

You can preview the large diff at https://github.com/BinaryAnalysisPlatform/FrontC/compare/master...jordr:FrontC:sourcesup?expand=1
Do you know what's going on / suggest a course of action?

**Comment by ivg** [comment]
We created this repository because the original authors removed it from the previous location (www.irit.fr) without notice and didn't reply to our emails. Possibly it was because the author changed the job and we were contacting him via the author's IRIT email. 

I don't think that at this time the projects are anyhow mergeable but if you're in contact with the author, it would be nice if you could introduce this project to him and ask him if he is interested in the merge.

**Comment by jordr** [comment]
I agree the repository/author can be hard to find/contact. It's unfortunate that you were not able to start from the more recent versions. I am in contact with Hugues Cassé and mentioned this project, but he would be too busy to merge or even continue maintaining his now-fork.

If a merge is not possible or a desirable use of anyone's time, it is at least interesting to be aware of both projects and later potentially cherry-pick features / replay commits if desired. Your branch is seemingly the better maintained one at this point.

**Comment by ivg** [comment]
Definitely, now if there will be any bug report, I would first look for a fix in that repository. It could be also possible that we can rebase our changes over their changes, but I doubt that it will succeed.
